app/app/src/main/kotlin/com/hedvig/app/feature/home/ui/HomeFragment.kt

```diff
@@ -231,6 +245,28 @@ class HomeFragment : Fragment() {
                 Spacer(Modifier.height(16.dp))
                 Spacer(Modifier.windowInsetsBottomHeight(WindowInsets.safeDrawing))
               }
+              Column {
+                TopAppBarWithActions {
+                  ToolbarChatIcon(
+                    onClick = { requireContext().startChat() },
+                  )
+                }
+                val shouldShowTooltip by produceState(false) {
+                  val currentEpochDay = LocalDate.now().toEpochDay()
```

Comment:
Could extract this logic to its own function

Reply:
Yeap, will do in another PR